src/itf/online-feature-itf.h

```diff
@@ -74,10 +74,15 @@ class OnlineFeatureInterface {
   /// the class.
   virtual void GetFrame(int32 frame, VectorBase<BaseFloat> *feat) = 0;
 
+  // Returns frame shift in seconds. Useful to support various time-based
+  // options like silence range to end the utterance.
```

Comment:
not sure what you mean by "silence range to end the utterance."  You mean silence duration?